refs/ietf.json

```diff
@@ -93929,7 +93929,7 @@
         "href": "https://datatracker.ietf.org/doc/html/rfc6173",
         "title": "Definitions of Managed Objects for the Internet Fibre Channel Protocol (iFCP)",
         "authors": [
-            "P. Venkatesen, Ed."
+            "P. Venkatesan, Ed."
```

Comment:
Some people's names are misspelled. Just because they're the editor of a document doesn't mean their name was spelled correctly.

In this case, I'm going with LinkedIn:

https://in.linkedin.com/in/prasanna-venkatesan-k-b5888a173